packages/gatsby/src/utils/webpack.config.js

```diff
@@ -527,16 +527,12 @@ module.exports = async (
           },
           priority: 30,
           minChunks: 1,
-          reuseExistingChunk: true,
```

Comment:
This doesn't make sense here as it would we weird that a user would have named a chunk the same as us